### ISSUE TITLE
Installer or something like this

### ISSUE THREAD

**Issue by komiljonov** [TheCherno/Hazel#643]
Hi devs. I cloned you repo and tryed install  your engine. But i can't. sometimes submodules not pulling. or libs not found. I used `git submodule update --init` it's not helped. 

So i think you must build and release your engini in github repo. Or add download section to your official website. With installer little bit ease for using your engine.

Sorry for bad english.

**Comment by VagueLobster** [comment]
> Hi devs. I cloned you repo and tryed install your engine. But i can't. sometimes submodules not pulling. or libs not found. I used `git submodule update --init` it's not helped.
> 
> So i think you must build and release your engini in github repo. Or add download section to your official website. With installer little bit ease for using your engine.
> 
> Sorry for bad english.

I could be wrong here, but it sounds like you didn't `git clone --recursive` which you need to do! Also the msdf-atlas-gen submodule is a little broken! You will need to change every `\\` in the .gitmodules file to `/` to be able to update it / download it👍

**Comment by komiljonov** [comment]
Yes you are right, msdf-atlas-gen module is broking in .gitmodules file. I replaced `\\` with `/` then it worked. I used engine. it needs too many features like undo and redo and etc.
